echo/README.md

```diff
@@ -1,5 +1,233 @@
 # Echo Example
 
-This example is designed to illustrate a few simple aspects of coding an actor module with wasmCloud. The first shows how to respond to simple HTTP requests, but this sample also shows how to use `serde` for JSON serialization.
+An actor is the smallest unit of deployable, portable compute within the wasmcloud ecosystem.
 
-This actor module responds to all incoming HTTP requests by returning a JSON object describing the inbound request.
+Actors are small WebAssembly modules that can handle messages delivered to them by the host runtime and can invoke functions on capability providers, provided they have been granted the appropriate privileges.
+
+Check out https://wasmcloud.dev/reference/host-runtime/actors/ for more.
+
+In this example our actor responds to all incoming HTTP requests by returning the HTTP request as a JSON object.
+
+We will have a two part tutorial, starting with running a precompiled actor stored in a remote repo, then we will write our own actor and swap it with the remote one. We will use a manifest file to run our actor declaratively via `wasmcloud`.
```

Comment:
s/repo/OCI registry (repo implies github when I think you want OCI registry here)